packages/infoportal-client/src/features/Meal/Pdm/Context/MealPdmContext.tsx

```diff
@@ -86,16 +88,39 @@ export const MealPdmProvider = ({children}: {children: ReactNode}) => {
           answers: record,
         })),
       ),
+      api.kobo.typedAnswers.search.meal_nfiPdm().then((_) =>
+        seq(_.data).map((record) => ({
+          type: PdmType.Nfi,
+          oblast: OblastIndex.byKoboName(record.oblast!)!.name,
+          project: DrcProjectHelper.search(record.donor),
+          office: fnSwitch(
+            record.office_responsible!,
+            {
+              dnk: DrcOffice.Dnipro,
+              hrk: DrcOffice.Kharkiv,
+              cej: DrcOffice.Chernihiv,
+              lwo: DrcOffice.Lviv,
+              umy: DrcOffice.Sumy,
+              nlv: DrcOffice.Mykolaiv
+            },
+            () => undefined,
```

Comment:
Breaking news! Now you can use `KoboXmlMapper.mapOffice(record.office_responsible)`